### PR TITLE
chore: add myself to usernames

### PR DESCRIPTION
**Pull request by prakamya-agrawal**

### CODE REVIEW

**Comment by henryiii** [comment]
Could you resolve conflicts, please?

**Comment by prakamya-agrawal** [comment]
> Could you resolve conflicts, please?

Hi Henry, 
I'm not sure what the conflict is. I'm able to click "Resolve Conflicts" and see the usernames.txt, but I'm not sure what it actually wants me to resolve.

**Comment by henryiii** [comment]
You need to come up with a merged version of the file manually. Generally, that means you remove the `<<<<<<`, `=====`, and `>>>>>` markers, merging the contents manually. In this case, it's just to include both parts alphabetically.

File: tests/usernames.txt [hunk 1]
```diff
@@ -4,5 +4,7 @@ bbixler
 cpappenheimer
 GuillermoFidalgo
 henryiii
+prakamya
+roy-cruz
 mthakorecern
-roy-cruz
\ No newline at end of file
+roy-cruz
```
Comment (henryiii) [inline tests/usernames.txt]: You left in the old "roy-cruz" when you moved it. I can fix it here, but something to watch for.

Comment (henryiii) [reply tests/usernames.txt]: Also, this isn't alphabetical.

**Pull request merged**